test/models/ReasonsRadioGroup.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from 'chai';
+
+import { createReasonsRadioGroup } from 'app/models/components/ReasonsRadioGroup';
+
+describe('The Reasons Radio Group factory', () => {
+    it('should only return radio buttons for enabled reasons', () => {
```

Comment:
One test feels bit light to be fair. What about env var? What about env var with more than one option?

Again this comment is only valid if you still to your approach. If you use 0/1 feature flag it might not be relevant anymore as most likely options would be hardcoded in NJK file.

Reply:
I'll look into additional test cases, unless we decide to change the feature flag approach.

Reply:
Resolved: Component factory has been removed as template is now static.